recipes/graphviz/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set version = "2.38.0" %}
+
+package:
+  name: graphviz
+  version: {{ version }}
+
+source:
+  fn:  graphviz-{{ version }}.tar.gz
+  url: http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz-{{ version }}.tar.gz
+  md5: 5b6a829b2ac94efcd5fa3c223ed6d3ae
+  patches:
+    - gvconfig_libdir.patch
```

Comment:
We should probably explain this patch. I know one of us got it from somewhere on the mailing listing, but it would be good if we could track this down and add the link. The patch is obviously correct, but it would just be nice to know when we can drop it.